config/m4/gdrcopy.m4

```diff
@@ -38,20 +39,26 @@ AS_IF([test "x$with_gdrcopy" != "xno"],
                             gdrcopy_happy="no"])
             ], [gdrcopy_happy="no"])
 
+        AS_IF([test "x$gdrcopy_happy" = "xyes"],
+            [AC_CHECK_LIB([gdrapi], [gdr_copy_to_mapping],
+                [GDR_DEFINES="-DUCT_CUDA_GDR_COPY_HAS_COPY_TO_MAPPING"],
+                [AC_MSG_NOTICE([gdrcopy using old API.])])])
+
         CFLAGS="$save_CFLAGS"
         CPPFLAGS="$save_CPPFLAGS"
         LDFLAGS="$save_LDFLAGS"
 
         AS_IF([test "x$gdrcopy_happy" = "xyes"],
             [
-                AC_SUBST(GDR_COPY_CPPFLAGS, "-I$ucx_check_gdrcopy_dir/include/ ")
+                AC_SUBST(GDR_COPY_CPPFLAGS, "-I$ucx_check_gdrcopy_dir/include/ $GDR_DEFINES")
```

Comment:
@yosefe is this ok? or should we go with AC_CHECK_DECLS(gdr_copy_to_mappin,...)

Reply:
AC_CHECK_DECLS is better